CONTRIBUTING.md

```diff
@@ -1,167 +1,113 @@
-﻿# CDNJS contributing document
+﻿# Contribute to CDNJS
 
- - Please read the instructions about sparseCheckout and shallow clone before any git clone operation to this huge repository: [documents/sparseCheckout.md](https://github.com/cdnjs/cdnjs/blob/master/documents/sparseCheckout.md), especially if you are using MacOS which uses case-insensitive filesystem by default, will bring you other issues.
- - Please make sure you are not going to submit a personal library, we have popularity requirement for the libraries, for the detail, see: [issue](#b-request-a-new-library)
+* Read the [sparseCheckout documentation](/documents/sparseCheckout.md) before you clone the CDNJS repo locally. It is not recommended to clone the whole repo as is due to its large size. Using sparse-checkout also works around issues on case-insensitive file systems (HFS+ on macOS).
+* All libraries on CDNJS must meet our [popularity criteria](#popularity-requirements-for-new-libraries).
 
 ## Table of Contents
 
-* Are you opening an issue? Please take a look at [issue](#a-issue) chapter.
+* [Issues](#issues)
+    * [General Conventions for Issues](#general-conventions-for-issues)
+    * [Popularity Requirements for New Libraries](#popularity-requirements-for-new-libraries)
+* [Pull Requests](#pull-requests)
+    * [Sparse Checkout and Shallow Clone](#sparse-checkout-and-shallow-clone)
+    * [General Conventions for Pull Requests](#general-conventions-for-pull-requests)
+    * [Update an Existing Library](#update-an-existing-library)
+    * [Add a New Library With Assets](#add-a-new-library-with-assets)
+    * [Add a New Library by a Single package.json file](#add-a-new-library-by-a-single-packagejson-file)
+    * [Pre-flight Checks](#pre-flight-checks)
+    * [Miscellaneous Information](#miscellaneous-information)
+### Issues
 
-* Are you creating a pull request? Please take a look at [pull request](#b-pull-request) chapter.
+#### General Conventions for Issues
 
- A. [issue](#a-issue)
+* [Search](https://github.com/cdnjs/cdnjs/issues/) for possible duplicates before creating a new issue.
+* If you're the author/maintainer of a library, prepend `[author]` to the title when creating issues about that library. Such issues will be processed with higher priority than others.
+* Tag related [people](https://github.com/blog/821-mention-somebody-they-re-notified), [issues](https://github.com/blog/957-introducing-issue-mentions) and commits while creating a new issue to make collaboration easier.
 
- * a. [general conventions](#a-general-conventions)
+#### Popularity Requirements for New Libraries
 
- * b. [request a new library](#b-request-a-new-library)
+A library must satisfy the following criteria to be eligible for CDNJS:
+* 100 stars on GitHub **or** 500 downloads in the last month on npm
+* Should not be a personal hobby project
+* Must have at least one official and publicly accessible repository
```

Comment:
@PeterDaveHello Do we request libs must be from `NPM Package` or `GitHub` for the auto-update mechanism?